jetty-io/src/main/java/org/eclipse/jetty/io/AbstractEndPoint.java

```diff
@@ -390,7 +390,7 @@ public FillInterest getFillInterest()
         return _fillInterest;
     }
 
-    protected WriteFlusher getWriteFlusher()
+    public WriteFlusher getWriteFlusher()
```

Comment:
This was made public just for a testcase?

Reply:
Yes. The correspondent `getFillInterest()` is already `public`.